### PR TITLE
feat: alert feature 1st commit

### PR DESCRIPTION
**Pull request by ankit9015**
Alert feature 1st commit:

It contains:
1. Universal .alert class for every alert block
2. Specific alert classes such as .alert-success, .alert-warning, .alert-error, etc.
3. Universal .alert-title class for the title of every alert block
alert-socketUI

### CODE REVIEW

File: alert.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.alert {
+    color: rgb(53, 52, 52);
+    font-family: "FontAwesome";
+    padding: 5px 10px;
+    /* box-shadow: 5px 10px; */
```
Comment (ajit-sharmaGH) [inline alert.css]: you can remove the commented code

File: alert.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.alert .alert-title {
+    display: inline;
+}
+
+/* ------------------------------- specific alerts ----------------------- */
+
+.alert-success {
+    background-color: rgb(133, 214, 133);
```
Comment (ajit-sharmaGH) [inline alert.css]: Use css variable colors with relative naming

**Review by ajit-sharmaGH — APPROVED** [review]
went through code, looks good, suggested some changes!

File: alert.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+.alert-error {
+    background-color: rgb(228, 88, 88);
+}
+
+.alert-warning {
+    background-color: rgb(236, 236, 82);
+}
+
+.alert-info {
+    background-color: rgb(92, 235, 235);
```
Comment (Nagashree01) [inline alert.css]: Use css variables here, will definitely save your time a lot

**Review by Nagashree01 — APPROVED** [review]

File: alert.css (hunk 1 above)
Comment (ankit9015) [reply alert.css]: I have done the suggested changes

File: alert.css (hunk 3 above)
Comment (ankit9015) [reply alert.css]: I have done the suggested changes

**Pull request merged**